Firestore/Example/SwiftBuildTest/main.swift

```diff
@@ -198,13 +198,13 @@ func readDocument(at docRef: DocumentReference) {
       if let data = document.data() {
         print("Read document: \(data)")
       }
-      if let data = document.data(with: SnapshotOptions.serverTimestampBehavior(.estimate)) {
+      if let data = document.data(with: .estimate) {
```

Comment:
Calling attention to this:

  * `data(with:)` is the default translation from the compiler
  * `data(serverTimestampBehavior:)` is what's called for in the spec (but was flagged by @ryanwilson)
  * `data(withServerTimestampBehavior:)` is another possible alternative

Reply:
We should resolve this as part of the API discussion, but I strongly feel that`data(with:.none`) is not the right path forward here :)

Reply:
Okay `data(with:)` is definitely out. I had previously added the `NS_SWIFT_NAME` to make it consistent with our current proposal. I'll update with the outcome of the API discussion.